exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/ResourceTranslator.java

```diff
@@ -17,13 +17,24 @@
 
 import com.google.api.MonitoredResource;
 import com.google.cloud.opentelemetry.resource.GcpResource;
+import com.google.common.base.Strings;
+import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.sdk.resources.Resource;
+import java.util.Objects;
+import java.util.Set;
+import java.util.logging.Logger;
 
 /** Translates from OpenTelemetry Resource into Google Cloud Monitoring's MonitoredResource. */
 public class ResourceTranslator {
+  private static final String CUSTOM_MR_KEY = "gcp.resource_type";
+  private static final String BLANK_STRING = " ";
```

Comment:
Should this be the empty string, or does it need to have a space?

Reply:
Discussed internally, this will not be required - only sending labels that are found and relying on back-end to reject malformed MR.